kernel/libc/math.c

```diff
@@ -12,25 +13,69 @@ static double reduce_angle(double angle)
 	return angle;
 }
 
+// Taylor series expansion for sine function
 double sin(double x)
 {
 	x = reduce_angle(x);
 
-	// Taylor series expansion for sine function
 	return x - (x * x * x) / 6.0 + (x * x * x * x * x) / 120.0 -
 	       (x * x * x * x * x * x * x) / 5040.0;
 }
 
+// Taylor series expansion for cosine
 double cos(double x)
 {
 	x = reduce_angle(x);
 
-	// Taylor series expansion for cosine function
 	return 1.0 - (x * x) / 2.0 + (x * x * x * x) / 24.0 -
 	       (x * x * x * x * x * x) / 720.0;
 }
 
+// Tangent calculation
 double tan(double x)
 {
 	return sin(x)/cos(x);
 }
+
+// Taylor series expansion for inverse sine function (arcsin)
+double arcsin(double x)
+{
+    return x + ( (exp(x, 3))/factorial(3) ) + (( 1 * 3 * exp(x, 5)) / 40.0 ) + ( ( 1 * 3 * 5 * exp(x, 7) ) / 384.0 );
```

Comment:
replace `factorial(3)` with 6

Reply:
Thanks brother, I'm going to fix it right now.